js/index.js

```diff
@@ -18,3 +19,77 @@ for (let i = 0; i < skills.length; i++) {
 
   skillsList.appendChild(skill);
 }
+
+// Lesson 4.3
+// Hides messages section if there are no messages
+displayMessages = document.getElementById("messages");
```

Comment:
Thank you for the feedback Tom! I have separated the event listeners and updated the button functions. The buttons now are hidden whenever an entry is being edited and the remove button no longer removes all of the entries. It appears that it was only removing all entries when you removed the very first entry. I believe this address most of the issues that you identified. I still see some areas for improvement. I would like to have each of the fields have labels when the different sections are being edited. Also, I think that the way I went about allowing the user to edit the email may cause issues if the user deletes the "mailto:" text. Lastly, I noticed that if I clicked anywhere in the messages section I was still getting some errors login into the console. Is this an issue with specificity since I have delegated my event listener to the parent element? If so, is there a way to be more specific with the event listener in this instance?

Thank you for all your help! It was cool to receive feedback on a project!